vscode/src/extension.ts

```diff
@@ -1,6 +1,6 @@
 import * as vscode from 'vscode';
-import { format } from 'sql-formatter';
-import type {
+import {
+  format,
```

Comment:
Initially made this change in vscode extension unintentionally with global search/replace.

It looked fine though, so decided to keep it. But do check this one more carefully.